File: ch13/Chapter13-WCFService-CS/App_Code/MyCustomDataContractService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.ServiceModel;
using System.Text;

public class MyCustomDataContractService : IMyCustomDataContractService
{
    public string HelloFirstName(Customer cust)
    {
        return "Hello " + cust.Firstname;
    }

    public string HelloFullName(Customer cust)
    {
        return "Hello " + cust.Firstname + " " + cust.Lastname;
    }
}

File: ch13/Chapter13-WCFService-VB/App_Code/MyCustomDataContractService.vb
Public Class MyCustomDataContractService
    Implements IMyCustomDataContractService

    Public Function HelloFirstName(cust As Customer) As String _
        Implements IMyCustomDataContractService.HelloFirstName
        Return "Hello " & cust.FirstName
    End Function

    Public Function HelloFullName(cust As Customer) As String _
        Implements IMyCustomDataContractService.HelloFullName
        Return "Hello " & cust.FirstName & " " & cust.LastName
    End Function
End Class